rpcs3/Emu/RSX/GL/GLPipelineCompiler.cpp

```diff
@@ -38,6 +38,10 @@ namespace gl
 
 	void pipe_compiler::operator()()
 	{
+#ifndef _WIN32
+		// Set low priority
+		thread_ctrl::scoped_priority low_prio(-1);
+#endif

```

Comment:
Are you sure these threads aren't used ingame?
Why decrease the priority?

Reply:
remnants from personal testing. removed.